package.json

```diff
@@ -23,7 +23,8 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "react-scripts build && npm run build:clean",
+    "build:clean": "cd build && cp static/js/main*.js main.js && cp static/css/main*.css main.css",
```

Comment:
I have but one question. If the clean line removes the giberish form the filenames, who changes the main.css and .js in the index.html?

Reply:
I just added a new script that copies the generated main js and css files but without the hash in the filename. this happens each time you do npm run build

Reply:
that i figured